silk/sql.py

```diff
@@ -34,10 +34,11 @@ def _explain_query(connection, q, params):
             # Work around some DB engines not supporting analyze option
             try:
                 prefix = connection.ops.explain_query_prefix(
-                    analyze=True
+                    analyze=True, **(SilkyConfig().SILKY_EXPLAIN_FLAGS or {})
                 )
             except ValueError as error:
-                if str(error) == "Unknown options: analyze":
+                error_str = str(error)
+                if error_str.startswith("Unknown options:") and "analyze" in error_str:
```

Comment:
When error occurred it would be better to show which option was a problem and append the error to the logger.

Reply:
Thanks, fixed